src/notification/index.js

```diff
@@ -77,6 +78,10 @@ export const getAccountFromNotificationData = (
       realm_uri,
       parsed_url: realmUrl,
       match_count: urlMatches.length,
+      known_urls: knownUrls,
```

Comment:
Does this really give us useful information for debugging this issue? This case is already all about when the URLs do match, so I'm not sure it does.

There is this that I mentioned above:

> I think in the current state of things (pre-#4817) we only count exact URL matches as matches, so we already know that these cases involve several `state.accounts` entries with the exact same URL. Post-#4817, it'd make sense to include in the logging a count of how many of the matches are exact matches.

But beyond that count, I don't think the actual array adds anything.

It's true that we already log this array in the no-matches case, a bit above this code. (And perhaps we should stop doing that.) Still, if we can avoid even having to give that explanation, that'd be better :slightly_smiling_face: